qprofiler/src/org/qcmg/qprofiler/messages.properties

```diff
@@ -1,28 +1,31 @@
-USAGE = usage: qprofiler [option...] -log logfile -loglevel INFO -output outputfile -input inputfile1 -input inputfile2 ... [-ntP 4 -ntC 16] [-exclude coverage,matrices,md,html] [-maxRecords 10000] [-tags XY,XZ,YZ]
+USAGE = usage: java -jar qprofiler.jar --input <input file1> --index <index file1> --input <input file2> --index <index file2>... --output <output xml file> --log <log file> [options]
+HELP_OPTION_DESCRIPTION = Show usage and help.
+VERSION_OPTION_DESCRIPTION = Show version number.
+INPUT_FILE_DESCRIPTION = Req, Input file in FASTQ, BAM/SAM or VCF format, also support FA, MA, QUAL, FASTA, GFF and GFF3 format.
+OUTPUT_FILE_DESCRIPTION = Opt, XML output which containing basic summary statistics. Def="qprofiler.xml".
+CONSUMER_THREADS_OPTION_DESCRIPTION = Opt, Number of threads to process inputed BAM records. Def=0.
```

Comment:
Is the default zero here?

Reply:
Indeed it is zero. but it is confusing, I added some comments into BamSummarizerMT

Reply:
I guess the question is, how many consumer threads will be used if this option is not supplied? Because the default behaviour is to run BamSummarizer, I would say that this should be 1.

Reply:
The BamSummarizer will be launched if this option is not supplied, no consumer threads will be launched. If any consumer threads require, the minimum value is one, then BamSummarizerMT will be launched.

Reply:
OK, perhaps should have the default `PRODUCER_THREADS_OPTION_DESCRIPTION `set to zero as well then. I find it confusing that the usage says it will default to 1 producer threads and 0 consumer threads